release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -705,11 +705,11 @@ The `oc adm upgrade` command now mentions any `Upgradeable=False` conditions, so
 === Networking
 
 [id="ocp-4-6-ovn-kubernetes-ga"]
-==== OVN-Kubernetes default Pod network provider GA
+==== OVN-Kubernetes cluster networking provider GA
 
-The OVN-Kubernetes Pod network provider is now GA. For more information, including details on feature parity with OpenShift SDN, refer to xref:../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[About the OVN-Kubernetes default Container Network Interface (CNI) network provider].
+The OVN-Kubernetes cluster networking provider is now GA. The networking provider is implemented as a Kubernetes CNI (Container Network Interface) plug-in. For more information, including details on feature parity with OpenShift SDN, refer to xref:../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[About the OVN-Kubernetes default Container Network Interface (CNI) network provider].
```

Comment:
For consistency with the link at the end of this release note (and the rest of the Release Notes assembly), I suggest flipping the acronym definition:

s/The networking provider is implemented as a Kubernetes CNI (Container Network Interface) plug-in./The networking provider is implemented as a Kubernetes Container Network Interface (CNI) plug-in./

Reply:
@lbarbeevargas, oh, I had it backwards. Definitely wasn't intentional.